doc/getting_started.md

```diff
@@ -0,0 +1,155 @@
+`data_steps`| number of steps (in integer format)
+`column_type`| "integer" or "date", default "integer"
+`minzoom`| minimum zoom level, optional
+`maxzoom`| max zoom level, optional
+`tiles`| tile array for this set, **required**
+`bounds`| [bounding box](http://wiki.openstreetmap.org/wiki/Bounding_Box) for tileset, optional
+
+#### Example of Metadata Document
+
+{% highlight js %}
+{
+  start: 0,
+  end: 100, 
+  resolution: 2
+  # scale: 1/resolution,
+  data_steps: 365,
+  column_type: "number"
+  "minzoom": 0,
+  "maxzoom": 11,
+  "tiles": [
+    'http://a.host.com/{z}/{x}/{y}.torque.json',
+    'http://b.host.com/{z}/{x}/{y}.torque.json',
+    'http://c.host.com/{z}/{x}/{y}.torque.json',
+    'http://d.host.com/{z}/{x}/{y}.torque.json'
+  ],
+  "bounds": [ -180, -85.05112877980659, 180, 85.0511287798066 ]
+}
+{% endhighlight %}
+
+### Tiles
+
+The TorqueMap Tiles document contains the required, core set of information to be rendered. This includes the number of pixels for the data, and the x and y values for each pixel.
+
+#### The URL Schema
+
+`http://host.com/{z}/{x}/{y}.torque.[json|bin]`
+
+#### Tile Format
+
+Each Torque tile is a JSON document containing an array, each of whose elements represents a point within the tile. The tile format is notated using the following format:
+
+Tile Format | Type | Description
+--- | --- |
+`x`| ` integer` | x pixel coordinate in tile system reference
+`y`| `integer` | y pixel coordinate in tile system reference
+`steps` | | time slots when this pixel is **active**, there is data at that time
+`values` | | values for each time slot<br /><br />**Tip:** You can use the values column to store encoding categories for your data
+
+#### Extracting Tile Results for Calculations
+
+You can extract the pixel position, and the current time, with the Tile document results.
+
+##### Extract the Pixel Position
+
+To extract the pixel position for the tile, use the following calculation:
+
+`pixel_x = x * resolution`
+`pixel_y = y * resolution`
+
+Where:
+
+`x` and `y` are in range [0, 256/resolution] to render the final pixel position (based on 256 x2 56 tiles).
+
+The coordinate origin for Torque tiles is the bottom left corner of the grid.
+
+##### Extract the Current Time
+
+To extract the current time from the Tile document, use the following calculation:
+
+`current_time = translate.start  + step * (translate.end - translate.start)/data_steps;`
+
+Where:
+
+The `current_time`, `translate.start, translate.end`, and `data_steps` values are used to extract the time.
+
+##### Tile Format Example
+
+{% highlight js %}
```

Comment:
Okay, I am not sure what this is as I simply followed the format of other examples. @matallo , can you advise?